account_invoice_pricelist_sale/tests/test_module.py

```diff
@@ -2,24 +2,21 @@
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo.tests.common import TransactionCase
+from odoo.addons.base.tests.common import BaseCommon
 
 
-class TestModule(TransactionCase):
+class TestModule(BaseCommon):
     def setUp(self):
         super().setUp()
         self.partner = self.env.ref("base.res_partner_12")
         self.product = self.env.ref("product.consu_delivery_01")
```

Comment:
please remove those demo records.
```suggestion
self.partner = self.env["res.partner"].create({"name": "Bob"})
self.product = self.env["product.product"].create({"name": "rubber band"})
```